app/lib/common/providers/room_providers.dart

```diff
@@ -326,6 +326,12 @@ final isDirectChatProvider =
   return convo?.isDm() == true && members.length == 2;
 });
 
+final isConvoBookmarked =
```

Comment:
room bookmarking would have been nicer but that would have required rust-sdk changes which I wanted to avoid for now. But it's easy to follow up on later with having this provider now.